lib/components/user/user-account-screen.js

```diff
@@ -0,0 +1,595 @@
+    ]
+
+    const {branding} = otpConfig
+
+    return (
+      <div class='otp'>
+        <Navbar fluid inverse>
+          <Navbar.Header>
+            <Navbar.Brand>
+              <div className='app-menu-container'>
+                <AppMenu />
+              </div>
+              <div
+                className={`icon-${branding}`}
+                // This style is applied here because it is only intended for
+                // desktop view.
+                style={{ marginLeft: 50 }} />
+            </Navbar.Brand>
+          </Navbar.Header>
+        </Navbar>
+
+        <form>
+
+          {isNewAccount
+            ? (
+              <div class='container'>
+                <h1>{realActivePane.title}</h1>
+                <div style={{minHeight: '20em'}}>
+                  {realActivePane.pane}
+                </div>
+
+                <FormGroup style={{padding: '20px 0px'}}>
+                  <nav aria-label='...'>
+                    {realActivePane.prevId &&
+                    <Button
+                      onClick={() => this._handleToPrevPane(panesForNewAccount)}
+                      style={{float: 'left'}}
+                    >
+                      Back
+                    </Button>
+                    }
+                    <Button
+                      bsStyle='primary'
+                      disabled={realActivePane.disableNext}
+                      onClick={() => this._handleToNextPane(panesForNewAccount)}
+                      style={{float: 'right'}}
+                    >
+                      {realActivePane.nextId ? 'Next' : 'Finish'}
+                    </Button>
+                  </nav>
+                </FormGroup>
+              </div>
+            )
+            : (
+              <div class='container'>
+                <h1>My Account</h1>
+                {
+                  panesForExistingAccount.map((pane, index) => (
+                    <div key={index} style={{borderBottom: '1px solid #c0c0c0'}}>
+                      <h3 style={{marginTop: '0.5em'}}>{pane.title}</h3>
+                      <div style={{marginLeft: '10%'}}>
+                        {pane.pane}
+                      </div>
+                    </div>
+                  ))
+                }
+
+                <FormGroup style={{padding: '20px 0px'}}>
+                  <nav aria-label='...'>
+                    <Button
+                      onClick={this._handleExit}
+                      style={{float: 'left'}}
+                    >
+                      Back
+                    </Button>
+                    <Button
+                      bsStyle='primary'
+                      onClick={this._handleExitAndSave}
+                      style={{float: 'right'}}
+                    >
+                      Save Preferences
+                    </Button>
+                  </nav>
+                </FormGroup>
+              </div>
+            )
+          }
+        </form>
```

Comment:
To improve readability, I recommend breaking up each suboption into its own render function. Ex:

```js
<form>
  {isNewAccount ? this._renderNewAccount() : this._renderExistingAccount()}
</form>
```